LightBulb/Views/Components/Settings/AdvancedSettingsTabView.axaml

```diff
@@ -18,11 +18,33 @@
                 HorizontalAlignment="Right"
                 VerticalAlignment="Center"
                 DockPanel.Dock="Right"
-                IsChecked="{Binding IsAutoStartEnabled}" />
+                IsChecked="{Binding IsAutoStartEnabled}"
+                Classes.accent="{DynamicResource UseAccentControls}" />
+        </DockPanel>
+
+        <!-- Theme -->
+        <DockPanel Margin="0,16,0,0" ToolTip.Tip="LightBulb UI Theme">
+            <TextBlock
+                VerticalAlignment="Center"
+                DockPanel.Dock="Left"
+                Text="Theme" />
+            <ComboBox
+                HorizontalAlignment="Right"
+                DockPanel.Dock="Right"
+                ItemsSource="{Binding ThemeArray}"
+                SelectedItem="{Binding Theme}"
+                Classes.accent="{DynamicResource UseAccentControls}">
```

Comment:
Let's not assign accent like that and instead due it either globally in the templates, or just keep to the default look and feel of the dark theme (without accents).

Reply:
As far as I can tell, there's no way to add a class globally, and Material.Avalonia uses classes exclusively to denote primary vs. accent theme colors. Is there a way to specify it in the template you're aware of? 

The primary color doesn't meet accessibility standards for control contrast against the backdrop, unfortunately. We'd have to change the primary color to something else, and then edit the sidebar/header/border/etc to specifically show a different color than primary while in dark mode. That seems like a more convoluted solution to me, but I guess I could try.

Reply:
Let's keep it this way for now, I think I'll try a better solution later